phys/module_sf_urban.F

```diff
@@ -783,6 +804,25 @@ SUBROUTINE urban(LSOLAR,                                           & ! L
    endif
    if(alhoption==1) ALH = ALH*alhdiuprf(tloc2)*alhseason(Kalh)
 
+   IF (distributed_aerodynamics_option) THEN
```

Comment:
also need to add a check for cases where distributed_aerodynamics_option is true but the input variables/data are not provided (i.e., MH_URB <=0 or LP_URB<=0).

Reply:
@cenlinhe  Within this if statement, some default values for the parameters when they are not available are set.
```fortran
ZR = MAX(MH_URB, 3.5)
Z0C = MAX(Z0_URB, 0.1)
lambda_p = MAX(0.05, MIN(1.0, LP_URB))
lambda_f = MAX(0.05, MIN(1.0, LF_URB_S))
```

Reply:
OK, I see.